notebook/i18n/fr_FR/LC_MESSAGES/nbui.po

```diff
@@ -0,0 +1,746 @@
+
+#: notebook/templates/notebook.html:186
+msgid "Insert Cell Below"
+msgstr "Insérer une Cellule en-Dessous"
+
+#: notebook/templates/notebook.html:189
+msgid "Cell"
+msgstr "Cellule"
+
+#: notebook/templates/notebook.html:191
+msgid "Run this cell, and move cursor to the next one"
+msgstr "Exécuter cette cellule, et déplacer le curseur à la suivante"
+
+#: notebook/templates/notebook.html:192
+msgid "Run Cells"
+msgstr "Exécuter les Cellules"
+
+#: notebook/templates/notebook.html:193
+msgid "Run this cell, select below"
+msgstr "Exécuter cette cellule et sélectionner la suivante"
+
+#: notebook/templates/notebook.html:194
+msgid "Run Cells and Select Below"
+msgstr "Exécuter les cellules et sélectionner celle en-dessous"
+
+#: notebook/templates/notebook.html:195
+msgid "Run this cell, insert below"
+msgstr "Exécuter la cellule et insérer après"
+
+#: notebook/templates/notebook.html:196
+msgid "Run Cells and Insert Below"
+msgstr "Exécuter les Cellules et insérer Après"
+
+#: notebook/templates/notebook.html:197
+msgid "Run all cells in the notebook"
+msgstr "Exécuter toutes les cellules du notebook"
+
+#: notebook/templates/notebook.html:198
+msgid "Run All"
+msgstr "Exécuter tout"
+
+#: notebook/templates/notebook.html:199
+msgid "Run all cells above (but not including) this cell"
+msgstr "Exécuter toutes les cellules au-dessus de celle-ci (non incluse)"
+
+#: notebook/templates/notebook.html:200
+msgid "Run All Above"
+msgstr "Exécuter au-dessus"
+
+#: notebook/templates/notebook.html:201
+msgid "Run this cell and all cells below it"
+msgstr "Exécuter cette cellule et celles en-dessous"
+
+#: notebook/templates/notebook.html:202
+msgid "Run All Below"
+msgstr "Exécuter en-dessous"
+
+#: notebook/templates/notebook.html:205
+msgid ""
+"All cells in the notebook have a cell type. By default, new cells are "
+"created as 'Code' cells"
+msgstr ""
+"Toutes les cellules dans le notebook sont caractérisée par un type de "
+"cellule. Par défaut, les nouvelles cellules sont de type « code »"
+
+#: notebook/templates/notebook.html:206
+msgid "Cell Type"
+msgstr "Type de Cellule"
+
+#: notebook/templates/notebook.html:209
+msgid ""
+"Contents will be sent to the kernel for execution, and output will display "
+"in the footer of cell"
+msgstr ""
+"Le contenu sera envoyé au noyau pour exécution, et la sortie sera affichée "
+"dans le pied de cellule"
+
+#: notebook/templates/notebook.html:212
+msgid "Contents will be rendered as HTML and serve as explanatory text"
+msgstr ""
+"Le contenu sera rendu en tant que HTML afin de servir de texte explicatif"
+
+#: notebook/templates/notebook.html:213 notebook/templates/notebook.html:298
+msgid "Markdown"
+msgstr "Markdown"
+
+#: notebook/templates/notebook.html:215
+msgid "Contents will pass through nbconvert unmodified"
+msgstr "Le contenu passera par nbconvert qui ne l'altèrera pas"
+
+#: notebook/templates/notebook.html:216
+msgid "Raw NBConvert"
+msgstr "Brute NBConvert"
```

Comment:
Hum, I'm sure we can find something better here.

Reply:
Simple?